adapters/Unity/UnityAdapter/GADMAdapterUnityConstants.h

```diff
@@ -14,6 +14,32 @@
 
 #import <Foundation/Foundation.h>
 
+typedef NS_ENUM(NSInteger, GADMAdapterUnityErrorCode) {
+  /// Invalid server parameters.
+  GADMAdapterUnityErrorInvalidServerParameters = 101,
+  /// Device not supported by UnityAds.
+  GADMAdapterUnityErrorDeviceNotSupported = 102,
+  /// UnityAds finished presenting with error state kUnityAdsFinishStateError.
+  GADMAdapterUnityErrorFinish = 103,
+  /// The Unity ad object is nil after calling its initializer.
+  GADMAdapterUnityErrorAdObjectNil = 104,
+  /// Failed to show Unity Ads due to ad not ready.
+  GADMAdapterUnityErrorShowAdNotReady = 105,
+  /// UnityAds called a placement changed callback with placement state
+  /// kUnityAdsPlacementStateNoFill.
+  GADMAdapterUnityErrorPlacementStateNoFill = 106,
+  /// UnityAds called a placement changed callback with placement state
+  /// kUnityAdsPlacementStateDisabled.
+  GADMAdapterUnityErrorPlacementStateDisabled = 107,
+  /// An ad was already loaded for this placement. UnityAds SDK does not support loading multiple
+  /// ads for the same placement.
+  GADMAdapterUnityErrorAdAlreadyLoaded = 108,
+  /// Banner size mismatch.
+  GADMAdapterUnityErrorSizeMismatch = 109,
+  /// UnityAds returned an initialization error
+  GADMAdapterUnityErrorAdInitializationFailure = 110
+};
+
```

Comment:
The error codes should remain in the GADMediationAdapterUnity.h file   Those are not constants plus  we have to keep  consistency throughout all the adapters.